src/components/ChoiceGroup/ChoiceGroup.test.tsx

```diff
@@ -0,0 +1,106 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import ChoiceGroup, { ChoiceGroupProps } from '.';
+// prettier-ignore
+import {
+  IChoiceGroupOptionProps,
+} from 'office-ui-fabric-react/lib/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.types';
```

Comment:
Without `prettier-ignore`, this gets put on one line and breaks our line-length rule. 
Let me know if you know a better way around it.

https://github.com/prettier/prettier/issues/2166

Reply:
odd

Reply:
Given https://github.com/OfficeDev/office-ui-fabric-react/blob/90eff99fab187ee41e325fca4d55d5c5fb838392/packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/index.ts 

you should be able to change it to the shorter:-

```ts
import { IChoiceGroupOptionProps } from 'office-ui-fabric-react/lib/components/ChoiceGroup/ChoiceGroupOption';
```